samples/graph-rsc/nodeJs/.env

```diff
@@ -1,3 +1,4 @@
 ClientId = 
 ClientSecret = 
-AllowedHosts = *
\ No newline at end of file
+HTTPS=true
```

Comment:
why do we need `HTTPS`?

Reply:
In local it was running HTTP, But in TTK ENDPOINT is running only through HTTPS and so we added.